### PR TITLE
Add constraints that account_number can't be an empty string

### PR DESCRIPTION
**Pull request by lavocatt**
See #240

### CODE REVIEW

**Review by Gundersanne — CHANGES_REQUESTED** [review]
Thanks! Some thoughts

File: internal/db/migrations/3_add_alter_table_composes.up.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE IF EXISTS composes ADD  CONSTRAINT PK_org_id CHECK ((account_number<>N''));
```
[the remark this answers is not in the document]
Comment (lavocatt) [reply internal/db/migrations/3_add_alter_table_composes.up.sql:1]: Oh yeah this, I really stack overflowed this one at it seemed to work so... gonna change the stuff you required.
I don't really know this much in sql btw

File: cmd/image-builder-db-test/main_test.go [hunk 2]
```diff
@@ -5,7 +5,7 @@ package main
 import (
 	"context"
 	"fmt"
-        "testing"
+	"testing"
```
[the remark this answers is not in the document]
Comment (lavocatt) [reply cmd/image-builder-db-test/main_test.go:8]: You can thank my vim plugin ^^ it does it automatically when I save a file :smile:

[the remark this answers is not in the document]
Comment (Gundersanne) [reply cmd/image-builder-db-test/main_test.go:8]: that is a good idea, I'll do that too :>

File: internal/tutils/idheader_utils.go [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+	"      \"is_internal\": true," +
+	"      \"locale\": \"en-US\"" +
+	"    }," +
+	"    \"internal\": {" +
+	"      \"org_id\": \"%s\"" +
+	"    }" +
+	"  }" +
+	"}"
+
+var noOrgIdIdHeader string = "{" +
```
[the remark this answers is not in the document]
Comment (lavocatt) [reply internal/tutils/idheader_utils.go]: Good idea!

**Review by Gundersanne — APPROVED** [review]
LGTM, thank you!

For future reference, `UnitTests:` in the commit topic can point to the name of the module, so in this case `v1`, or `server`.

**Pull request merged**